src/DotVVM.Framework/Resources/Scripts/spa/spa.ts

```diff
@@ -8,27 +8,28 @@ import { DotvvmPostbackError } from '../shared-classes';
 export const isSpaReady = ko.observable(false);
 
 export function init(): void {
-    const spaPlaceHolder = getSpaPlaceHolder();
-    if (spaPlaceHolder == null) {
-        throw new Error("The SpaContentPlaceHolder control was not found!");
+    const spaPlaceHolders = getSpaPlaceHolders();
+    if (!spaPlaceHolders) {
```

Comment:
Negation of empty array is always false, we should check for `...length == 0` (or `!...length`)

Reply:
Fixed